docs/source/documentation/user-management.erb

```diff
@@ -0,0 +1,58 @@
+---
+title: User Management
+---
+
```

Comment:
There should be a H1 here to populate the rendered page title:
`# User Management`